lib/venue.rb

```diff
@@ -1,2 +1,24 @@
 class Venue
+  attr_reader :name,
+              :capacity,
+              :patrons
+
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+    @patrons = []
+  end
+
+  def add_patron(add)
+     @patrons.push add
```

Comment:
Pay attention to the interaction pattern here. `add_patron` takes an argument of a `patron` not `add`. So it would look like:
```
def add_patron(patron)
  @patrons.push(patron)
end
```